Sources/XiEditor/MarkerBar.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Cocoa
+
+class Marker {
+    let line: Int
+    let color: NSColor
+
+    init(_ line: Int, color: NSColor) {
+        self.line = line
+        self.color = color
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
```

Comment:
Hmm do we need this init? This class does not inherit from anything, my guess would be that it's not needed.

Reply:
True, will remove.